test/jdk/javax/swing/JTabbedPane/4209065/bug4209065.java

```diff
@@ -21,37 +21,65 @@
  * questions.
  */
 
-import java.awt.EventQueue;
+/*
+ * @test
+ * @bug 4209065
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @summary To test if the style of the text on the tab matches the description.
+ * @run main/manual bug4209065
+ */
+
 import java.lang.reflect.InvocationTargetException;
 
-import javax.swing.JApplet;
+import javax.swing.JFrame;
 import javax.swing.JLabel;
 import javax.swing.JTabbedPane;
+import javax.swing.SwingUtilities;
 
-/**
- * @test
- * @bug 4209065
- * @author Georges Saab
- * @run applet/manual=yesno bug4209065.html
- */
-public final class bug4209065 extends JApplet {
-
-    @Override
-    public void init() {
-        try {
-            EventQueue.invokeAndWait(this::createTabbedPane);
-        } catch (InterruptedException | InvocationTargetException e) {
-            throw new RuntimeException(e);
-        }
+
+public final class bug4209065 {
+
+    private static JFrame frame;
+    private static final String text = """
+            If the style of the text on the tabs matches the descriptions,
+             press PASS.
+
+            NOTE: where a large font is used, the text may be larger than the
+             tab height but this is OK and NOT a failure.
+            """;
+
```

Comment:
looks ok in jtreg but standalone execution in my windows fails with this String initialization. Probably we can do normal string with "+" as continuation instead of """ literal
```

javac bug4209065.java
bug4209065.java:44: error: unclosed string literal
    private static final String text = """

```

Reply:
SInce you are hardcoding PassFailFrame column to 35, it seems some word break is not perfect in the above instructions.
Probably, you can put \n break at appropriate positions in NOTE to show better.